leenlab2/reminiscence
# Pause menu 2

# Testing
- Press ESC or Left Shoulder to pause
- Check navigation works between buttons
- Pressing resume should bring you back to the game
- Pressing main menu should reset the game
- Press settings to get settings page. You can exit by clicking on rreturn (which is selected by default)
